### PR TITLE
Add patches API Spec

### PR DESCRIPTION
**Pull request by humulla**

### CODE REVIEW

File: docs/patches-api.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Following that, each individual service can use this data to develop their custom business logic. 
+
+
+
+## API Endpoints:
+
+GET /patch/{id}
+GET /patch?parentId={parentId}
+DELETE /patch{id}/responsible-entity/{responsible-entity-id}
```
Comment (LBHSPreston) [inline docs/patches-api.md]: Is there a missing '/' here /patch**/**{id}/?

File: docs/patches-api.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+POST /PATCH
+PATCH /patch{id}/responsible-entity/{responsible-entity-id}
```
Comment (LBHSPreston) [inline docs/patches-api.md]: Is there a missing '/' here /patch**/**{id}/?

Comment (humulla) [reply docs/patches-api.md]: Good call

File: docs/patches-api.md (hunk 1 above)
Comment (humulla) [reply docs/patches-api.md]: Good call

**Review by LBHSPreston — APPROVED** [review]

File: docs/patches-api.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+For this reason, we propose a single, simple data entity to be created to contain a list of all patches and areas we have in the various services, as well as a reference to the responsible entity.
+
+Following that, each individual service can use this data to develop their custom business logic. 
+
+
+
+## API Endpoints:
+
+GET /patch/{id}
```
Comment (amidomichael) [inline docs/patches-api.md:58]: looks like the formatting of these in the markdown file is a bit off

File: docs/patches-api.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+## Vision
+-   Single, centralized data store for our list of patches and areas,   regardless of which domain they apply to.
+-   Single, reusable microservice that manages our areas and patches data, including reassignment. 
+-   Microservice and data domain designed with extensibility in mind, so future service areas could potentially re-use it and extend it if / when future user needs are identified
+
+
+## User needs
+-   As a MMH user, I want to be able to see which patch is responsible for a property.
+-   As a MMH user (manager), I want to be able to re-assign the responsible officer/s for a given patch.
+-   As a MMH user, I want to know which officer/s is/are responsible for a patch or area.
```
Comment (rashmishetty1011) [inline docs/patches-api.md:18]: Can you please expand on the so that sections please for all user needs.

File: docs/patches-api.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+-   As a MA user, I want to know which cases are allocated to the patch I am assigned to so that I can focus only on the cases that are relevant to me.
+-   As a MA manager I want to be able to easily assign arrears cases to a group of workers so I can manage casework effectively.
+
+## Workshop outcome
+
+
+We looked at two services - Manage a Tenancy (MaT) and Manage Areas(MA) and concluded that patches and areas are different in the context of the two services. Although they do refer to geographical areas, those areas are different based on the service. Further to this, in the context of MA, patches for leaseholders are different - not based only on a geographical area. Reassignment is also different between the two applications. With MaT, reassignment of the responsible officer/s for a patch happens when someone is off sick, on maternity leave, etc. Once a patch is reassigned, the responsible officer would see all processes/action items that are related to their patch in their work tray. With MA, everyone can see all items in their work tray, which leads towards the concept of a “group” responsible for a patch, rather than individuals.
```
Comment (rashmishetty1011) [inline docs/patches-api.md]: Is there any more differences between the two services. If possible please use the correct format to note down the differences

File: docs/patches-api.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+However, despite the different business processes of dealing with a patch, the data we store for a patch area at its core is the same.
+Patch/area code/name
+Responsible entity (whether that’s an individual or a group)
+
+For this reason, we propose a single, simple data entity to be created to contain a list of all patches and areas we have in the various services, as well as a reference to the responsible entity.
```
Comment (rashmishetty1011) [inline docs/patches-api.md]: Please change to "Due to above reasons we propose a single...."

**Review by rashmishetty1011 — APPROVED** [review]

**Pull request merged**